stock_picking_reorder_lines/views/view.xml

```diff
@@ -3,7 +3,6 @@
     <data>
 
         <record id="view_move_picking_tree" model="ir.ui.view">
-            <field name="name">stock.move.tree</field>
```

Comment:
@pedrobaeza @yvaucher : it is a useless field.
And it will be generated automatically when Odoo load views.
if model = 'stock.move' and type = 'tree' then name = 'stock.move tree'.
Again, I don't understand the interest to set manually another value. Please indicate what is the bad consequencies.

(FYI, I shared like 10 / 15 modules in OCA, and none has this field defined...)